app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -354,12 +354,12 @@ public void showDuplicatePicturePopup() {
                 String.format(Locale.getDefault(),
                         uploadTitleFormat,
                         uploadItem.getFileName()),
-                () -> {
-
-                },
                 () -> {
                     uploadItem.setImageQuality(ImageUtils.IMAGE_KEEP);
                     onNextButtonClicked();
+                },
+                () -> {
```

Comment:
the DialogUtil class supports passing null for a click action, I think that would be better than taking up 3 lines

Reply:
> the DialogUtil class supports passing null for a click action, I think that would be better than taking up 3 lines

Got it!